CONTRIBUTING.md

```diff
@@ -28,6 +28,19 @@ functionality, also via pull requests.
 Please consult the [ArviZ documentation](https://arviz-devs.github.io/arviz/)
 to ensure that any new contribution does not strongly overlap with existing functionality.
 
+## Steps before starting work
+Before starting a work on a pull request double check that no one else is working on the ticket in both issue tickets and pull requests.
+
+### If an issue ticket exists 
+If an issue exists check the ticket to ensure no one else has started work. If first to start work, comment on the ticket to make it evident to others. If the comment looks old or abandoned leave a comment asking if you may start work.
+
+### If an issue ticket doesn't exist
+Open an issue ticket for the issue and state that you'll be solving the issue with a pull request. Optionally create a pull request and add `[WIP]` in the title to indicate Work in Progress.
+
+### In the event of a conflict
+In the event of two or more people working on the same issue, the general precedence will go to the person who first commented in the issue. If no comments it will go to the first person to submit a PR for review. Each situation will differ though the core contributors will make the best judgement call in each situation.
```

Comment:
I think a word or punctuation sign is missing here: "Each situation will differ though the core contributors will make the best judgement call in each situation."

Reply:
Thanks! Reworded